UPGRADE-3.x.md

```diff
@@ -4,6 +4,8 @@ UPGRADE 3.x
 UPGRADE FROM 3.0 to 3.1
 =======================
 
+## Fix products collection nagivation
```

Comment:
Either remove this headlines or add some more information to this.

Reply:
Also, "navigation" is spelt wrong :P

Reply:
Okay guys i'm not english fluent ^^

Reply:
@core23 is it better ?